### ISSUE TITLE
The title:afters load in fairly slowly

### ISSUE THREAD

**Issue by j-frost** [AdventureLookup/AdventureLookup#263]
I'm talking about these:
![image](https://user-images.githubusercontent.com/6002677/37315721-5e949aa2-265b-11e8-9d44-8e2ccfe25fc7.png)
Looks like they're from a custom font? They load in quite slowly for me, both on localhost and dev.adventurelookup. Do we have some superfast CDN for prod or is this something we need to look at? 

For reference, the download queues at around ~560ms, and finishes ~190-360ms after that. That's noticeable (especially because the replacement is some grand old Wingdings style boxes) and not entirely warranted.

**Comment by cmfcmf** [comment]
Good catch!
These are FontAwesome 4 icons. Prod uses the same infrastructure as dev.adventurelookup.com, so there shouldn't be noticable differences. 
We should configure nginx to serve files inside the `web/assets/` with never expiring cache headers (whenever we make changes to our CSS/JS files, webpack will generate new unique filenames). I'll try this on the dev server, Jerry will have to do it on prod.

**Comment by cmfcmf** [comment]
I've now added caching of the assets folder to the nginx config on dev.adventurelookup.com:
```nginx
# ...
        location /assets {
                expires max;
                add_header Pragma public;
                add_header Cache-Control "public";
        }
# ...
```

This made the page load quite a bit faster for me, because all CSS/JS files as well as FontAwesome are now directly loaded from browser cache. Could you please confirm, @j-frost?

**Comment by j-frost** [comment]
It did make it load faster, but the difference is still noticeable, probably mostly because they're loaded as boxes first and then replaced with the font contents. Is it possible for us to find Unicode equivalents while loading?

**Comment by cmfcmf** [comment]
I'll look into it.

**Comment by cmfcmf** [comment]
I think I improved caching even more with #265. This should be sufficient then 🚀

**Comment by cmfcmf** [comment]
@JohnnyFlash has also added the nginx rule to the prod server. The font is loading a bit slower there though, because it is still using `font-awesome-webpack`. Once `dev` is merged into `master`, there shouldn't be any flickering on `master` either.
